pkgs/applications/networking/browsers/firefox/wrapper.nix

```
@@ -2,7 +2,7 @@
   stdenv,
   lib,
   makeDesktopItem,
-  makeWrapper,
+  makeWrapper, # don't switch to makeBinaryWrapper until it supports all flags used, e.g. --run
```

```suggestion
  makeShellWrapper,
```
an explicit `makeShellWrapper` is the better idea IMO